demo/shaders/glsl/mesh_shadow_map.glsl

```diff
@@ -0,0 +1,40 @@
+// @[export]
+// @[internal_buffer]
+layout (set = 0, binding = 0) uniform PerViewData {
+    mat4 view;
+    mat4 view_proj;
+} per_view_data;
+
+// @[immutable_samplers([
+//     (
+//         mag_filter: Linear,
+//         min_filter: Linear,
+//         mip_map_mode: Linear,
+//         address_mode_u: Repeat,
+//         address_mode_v: Repeat,
+//         address_mode_w: Repeat,
+//         max_anisotropy: 16.0,
+//     )
+// ])]
+layout (set = 0, binding = 1) uniform sampler smp;
```

Comment:
It's not clear to me if this needs to be defined for this shadow. We only bind a depth attachment, right?

Reply:
Yeah, I don't think we would need that now

Reply:
Removed.